src/main/drivers/serial.h

```diff
@@ -31,8 +31,8 @@ typedef enum {
     SERIAL_INVERTED      = 1 << 0,
     SERIAL_STOPBITS_1    = 0 << 1,
     SERIAL_STOPBITS_2    = 1 << 1,
-    SERIAL_PARITY_NO     = 0 << 2,
-    SERIAL_PARITY_EVEN   = 1 << 2,
+    UNUSED_1             = 0 << 2,
```

Comment:
I was a bit afraid of moving settings at bits 3 and 4, maybe I should have just moved them one bit up and allocated bits 2 and 3 for parity values.